src/__tests__/tigris.rpc.spec.ts

```diff
@@ -773,12 +773,15 @@ describe("rpc tests", () => {
 		await c1.set("k4", { a: "b", n: 12 });
 		expect((await c1.get("k4")).value).toEqual({ a: "b", n: 12 });
 
-		const keys = await c1.keys();
+		const keysArrays = await c1.keys().toArray();
+		const keys: Array<string> = new Array<string>();
+		keysArrays.forEach((keysArray) => keysArray.forEach((key) => keys.push(key)));
+
+		expect(keys).toHaveLength(4);
```

Comment:
nit: tests are doing a lot. Maybe break them down as individual tests of set/del etc based on apis:

```
describe("cache tests", () => {
    it("set()") {}
    it("del()") {}
    it("keys()") {}
   ....
})
```

Reply:
they will go in bulk and repeat things if we break it down. You still need to exercise set and keys/get to test del. So keeping it under crud umbrella.